src/styles/components/button.scss

```diff
@@ -476,6 +476,7 @@ $btn-dark-outline-hover-bg-color: var(--btn-dark-outline-hover-bg-color);
 .btn-group {
 	&>.btn {
 		flex: none;
+		z-index: 0 !important;
```

Comment:
@Aringoaway Are you sure about `z-index: 0` ?

Reply:
I have now tested these changes and decided to remove this branch completely, because removing the `height` in [this request](http://gitlab.teskalabs.int/asab/asab-library-webui/-/merge_requests/31) completely solves the problem